warehouse/warehouse.go

```diff
@@ -367,8 +367,8 @@ func (wh *HandleT) initTableUploads(upload warehouseutils.UploadT, schema map[st
 }
 
 func (wh *HandleT) initUpload(warehouse warehouseutils.WarehouseT, jsonUploadsList []*StagingFileT) warehouseutils.UploadT {
-	sqlStatement := fmt.Sprintf(`INSERT INTO %s (source_id, namespace, destination_id, destination_type, start_staging_file_id, end_staging_file_id, start_load_file_id, end_load_file_id, status, schema, error, first_event_at, last_event_at, created_at, updated_at, timings)
```

Comment:
Why is timing initialisation removed ?

Reply:
added initially to avoid error in sql Scan(). Since it is handled now, did not initialize it